standalone/src/models/store/store/store-draft/presets/empty.ts

```diff
@@ -1,8 +1,9 @@
-import { TStoreDraftBuilder } from '../../types';
-import StoreDraft from '../builder';
+import { TBuilder } from '@/core';
+import { TStoreDraftGraphql, TStoreDraftRest } from '../../types';
+import { StoreDraftGraphql, StoreDraftRest, StoreDraft } from '../index';
 
-const empty = (): TStoreDraftBuilder =>
-  StoreDraft()
+export const emptyDraftRest = (): TBuilder<TStoreDraftRest> =>
```

Comment:
We should export the same named constants as in other files for consistency.
Please take a look at [this file](https://github.com/commercetools/test-data/blob/main/standalone/src/models/cart/cart/line-item/line-item-draft/presets/empty.ts) as an example.